common/persistence/sql/sqlTaskManager.go

```diff
@@ -72,15 +73,23 @@ func (m *sqlTaskManager) LeaseTaskList(request *persistence.LeaseTaskListRequest
 		TaskType: common.Int64Ptr(int64(request.TaskType))})
 	if err != nil {
 		if err == sql.ErrNoRows {
+			tlInfo := &sqlblobs.TaskListInfo{
+				AckLevel:         &ackLevel,
+				Kind:             common.Int16Ptr(int16(request.TaskListKind)),
+				ExpiryTimeNanos:  common.Int64Ptr(0),
+				LastUpdatedNanos: common.Int64Ptr(time.Now().UnixNano()),
```

Comment:
Could we have something like timeNowPtr()?

Reply:
done